pymeasure/instruments/anritsu/anritsuMS2090A.py

```diff
@@ -25,7 +25,8 @@
 from pymeasure.instruments import Instrument
 from pymeasure.instruments.validators import (
     strict_discrete_set,
-    truncated_range
+    truncated_range,
+    strict_range

```

Comment:
Just an information: In such a case it is good to add a comma behind strict_range as well, i.e. just add a comma behind every item, even the last one.
That way you can add or remove elements (by removing their line) in that list without affecting other lines (as you now changed line 28 in order to add line 29).